.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
+orbs:
+  helm: circleci/helm@1.1.2
+jobs:
+  build:
+    machine:
+      image: ubuntu-1604:201903-01
+      # need to upgrade plan before using large executor
+      resource_class: large
+    working_directory: ~/go/src/github.com/solo-io/gloo-mesh-enterprise-helm
+    steps:
+      - checkout
+      - helm/install-helm-client:
+          version: v3.2.4
+      - run:
+          name: install go
```

Comment:
(not needed for this PR) we could eventually leverage [Circle's caching feature](https://circleci.com/docs/2.0/caching/) to save some compute in these setup steps